Docs/Swinject.md

```diff
@@ -0,0 +1,3 @@
+# Swinject
+
+[Swinject](https://github.com/Swinject/Swinject/) and [SwinjectAutoregistration](https://github.com/Swinject/SwinjectAutoregistration) have been copied into the Knit project in order to allow deviation from the Swinject project which is not being well maintained.
```

Comment:
Can we include the source commits from those projects?

Reply:
Good idea, listed the source commits below.